github/libp2p.yml

```diff
@@ -5091,13 +5087,14 @@ teams:
       # INFO: There are others who could certainly qualify to be members of this team.  
       # There is a balance to be had to ensure there are enough knowledgeable people available to support the needs/requests of the github org,
       # and reducing risk by not having too many with the escalation path that this role affords.
-      maintainer:
-        - dhuseby
       member:
         # Why @achingbrain?
         # 1. Long-time and active libp2p maintainer, especially to js-libp2p.
         # 2. JS leader at IP Shipyard, which is an organization receiving significant grant funding for libp2p development and maintenance.
         - achingbrain
+        # Why @dhuseby?
+        # 1. Project community leader interfacing with multiple implementation teams.
+        - dhuseby
```

Comment:
Grabbed text from this commit https://github.com/libp2p/github-mgmt/commit/d2487f196b683787d22b62101e2317f474f29707, which is effectively being reverted here